util/plancodec/id.go

```diff
@@ -92,8 +92,8 @@ const (
 	TypeWindow = "Window"
 	// TypeShuffle is the type of Shuffle.
 	TypeShuffle = "Shuffle"
-	// TypeShuffleDataSourceStub is the type of Shuffle.
-	TypeShuffleDataSourceStub = "ShuffleDataSourceStub"
+	// TypeShuffleReceiverStub is the type of Shuffle.
+	TypeShuffleReceiverStub = "ShuffleReceiverStub"
```

Comment:
How about removing `Stub` and using `ShuffleReceiver`?

Reply:
Good advice, accepted.